packages/lit-html/src/lit-html.ts

```diff
@@ -189,16 +189,19 @@ interface DebugLoggingWindow {
  * Compiled out of prod mode builds.
  */
 const debugLogEvent = DEV_MODE
-  ? (event: LitUnstable.DebugLog.Entry) => {
+  ? () => {
       const shouldEmit = (window as unknown as DebugLoggingWindow)
         .emitLitDebugLogEvents;
       if (shouldEmit) {
+        return undefined;
+      }
+      return (event: LitUnstable.DebugLog.Entry) => {
```

Comment:
Do you think it's that much better to be lazier here rather than removing the clone() calls? Why does the element need to be cloned in the first place? I think that could still fail in some cases when the event are enabled.